lib/ruby_lsp/requests/on_type_formatting.rb

```diff
@@ -185,6 +187,31 @@ def find_indentation(line)
 
         count
       end
+
+      def auto_indent_after_end_keyword
```

Comment:
You still need the Sorbet signature though. Typechecking won't pass without. It should be `sig { void }`.